.php-cs-fixer.dist.php

```diff
@@ -15,9 +15,8 @@
         'void_return' => true,
         'native_function_invocation' => ['include' => ['@compiler_optimized'], 'scope' => 'namespaced'],
         'php_unit_test_case_static_method_calls' => ['call_type' => 'self'],
-        'php_unit_strict' => true,
-        // @todo: when we'll support only PHP 8.0 and upper, we can enable `parameters` for `trailing_comma_in_multiline` rule
-        'trailing_comma_in_multiline' => ['after_heredoc' => true, 'elements' => ['array_destructuring', 'arrays', 'match'/* , 'parameters' */]],
+        'php_unit_strict' => false,
```

Comment:
Why changing this?

Reply:
Because it converts some call from `self::assertEquals` to `self::assertSame` and then the tests break